schedule/yam/test_cases/guided_ext4_s390x_zvm.yaml

```diff
@@ -13,5 +12,3 @@ schedule:
   system_validation:
     - console/validate_partition_table_via_parted
```

Comment:
we will have a way to unify this as result of https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19634/files#r1662166471